### PR TITLE
add remote image support

### LINKED ISSUE #2: remote image support
I'd love to be able to download and set a remote desktop image.

`wallpaper http://imgur.com/a/DdV4w#VGgJD9O`

### PR DESCRIPTION
**Pull request by rawgni**
To fix https://github.com/sindresorhus/wallpaper-cli/issues/2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,10 @@
     "image"
   ],
   "dependencies": {
+    "is-url-superb": "^2.0.0",
     "meow": "^3.4.2",
+    "path": "^0.12.7",
```
Comment (sindresorhus) [inline package.json]: `path` is a core module. You can remove this.

**Comment by sindresorhus** [comment]
Nice! Can you fix the lint issues https://travis-ci.org/sindresorhus/wallpaper-cli/builds/125417553 and update the CLI help to show that it supports and URL too? :)

File: cli.js [hunk 2]
```diff
@@ -1,24 +1,40 @@
...
 if (input) {
-	wallpaper.set(input);
+	if (isUrl(input)) {
+		const file = tempfile(path.extname(input));
+		got
+			.stream(input)
+			.pipe(fs.createWriteStream(file))
+			.on('finish', () => {
+				wallpaper.set(file);
+			});
```
Comment (kevva) [inline cli.js:34]: Could do `.on('finish', () => wallpaper.set(file))` if we want to make it even smaller :).

Comment (sindresorhus) [inline cli.js:34]: @kevva I try not to do that with code that doesn't return as it makes the code harder to read.

File: cli.js [hunk 3]
```diff
@@ -1,24 +1,40 @@
...
 const meow = require('meow');
 const wallpaper = require('wallpaper');
+const isUrl = require('is-url-superb');
+const got = require('got');
+const tempfile = require('tempfile');
 
 const cli = meow(`
 	Usage
-	  $ wallpaper [file]
+	  $ wallpaper [file | url]
```
Comment (sindresorhus) [inline cli.js:13]: This needs to be reflected in the readme too.

File: cli.js [hunk 4]
```diff
@@ -1,24 +1,40 @@
...
 
 	Example
 	  $ wallpaper unicorn.jpg
 	  $ wallpaper
 	  /Users/sindresorhus/unicorn.jpg
+	  $ wallpaper https://assets-cdn.github.com/images/modules/open_graph/github-octocat.png
```
Comment (sindresorhus) [inline cli.js]: Can you find a shorter example URL?

Comment (rawgni) [inline cli.js]: will this work ? https://octodex.github.com/images/dojocat.jpg

Comment (sindresorhus) [inline cli.js]: 👍

**Comment by sindresorhus** [comment]
Nice work @rawgni :)

**Pull request closed without merging**